### ISSUE TITLE
fancontrol crashes with: Couldn't terminate ADLX

### ISSUE THREAD

**Issue by hmsta** [Rem0o/FanControl.ADLX#43]
every morning when I wake up my windows 10 pc from sleep fancontrol is gone. 

I checked the log and it looks to me that the ADLX plugin is causing a crash when the PC wakes up?

```
17.05.2024 10:00:36: ADLXWrapper.ADLXEception: Result: ADLX_ORPHAN_OBJECTS 
Couldn't terminate ADLX
   bei ADLXWrapper.Extensions.ThrowIfError(ADLX_RESULT result, String message)
   bei ADLXWrapper.ADLXWrapper.Dispose()
   bei FanControl.ADLX.ADLXPlugin.Close()
   bei FanControl.Domain.ComputerAccessLayer.Close()
18.05.2024 09:57:33: ADLXWrapper.ADLXEception: Result: ADLX_ORPHAN_OBJECTS 
Couldn't terminate ADLX
   bei ADLXWrapper.Extensions.ThrowIfError(ADLX_RESULT result, String message)
   bei ADLXWrapper.ADLXWrapper.Dispose()
   bei FanControl.ADLX.ADLXPlugin.Close()
   bei FanControl.Domain.ComputerAccessLayer.Close()
19.05.2024 09:44:41: ADLXWrapper.ADLXEception: Result: ADLX_ORPHAN_OBJECTS 
Couldn't terminate ADLX
   bei ADLXWrapper.Extensions.ThrowIfError(ADLX_RESULT result, String message)
   bei ADLXWrapper.ADLXWrapper.Dispose()
   bei FanControl.ADLX.ADLXPlugin.Close()
   bei FanControl.Domain.ComputerAccessLayer.Close()
20.05.2024 10:10:02: ADLXWrapper.ADLXEception: Result: ADLX_ORPHAN_OBJECTS 
Couldn't terminate ADLX
   bei ADLXWrapper.Extensions.ThrowIfError(ADLX_RESULT result, String message)
   bei ADLXWrapper.ADLXWrapper.Dispose()
   bei FanControl.ADLX.ADLXPlugin.Close()
   bei FanControl.Domain.ComputerAccessLayer.Close()
```

**Comment by Rem0o** [comment]
What version of FanControl and are you using the latest build of the plugin?

**Comment by Rem0o** [comment]
[FanControl.zip](https://github.com/Rem0o/FanControl.ADLX/files/15395900/FanControl.zip)
I compiled a fix, replace your exe/dll with this one. Either net8 or 4.8, depending on the version you got.

Tell me if the problem occurs (or not) in the next few days.

**Comment by Rem0o** [comment]
Sanity check, if you hit ctrl r in the app multiple times in a row, any error at all?

**Comment by burkm489** [comment]
This same problem seems to appear on my system. I get ADLX errors "en mass", which finally closed the log file on 19.10.2025, but reopened shortly on 10.12.2025 with (probably) the same error message in the FanControl log file. Nothing has been written again afterwards. My FanControl version is right now V251...
